module/zstd/zfs_zstd.c

```diff
@@ -688,6 +688,19 @@ zstd_mempool_deinit(void)
 	zstd_mempool_cctx = NULL;
 }
 
+/* release unused orphan memory from pool */
+
+void
+zfs_zstd_cache_reap_now(void)
+{
+	/*
+	 * calling alloc with zero size seeks
+	 * and releases orphan objects
```

Comment:
I don't think "orphan objects" is right phrase to use here since it implies their lost.  How about "unused objected" and moving this all to the block comment above the function.

```c
/*
 * Releases unused memory from the pool by calling
 * zstd_mempool_alloc() with a zero size.
 */
void
zfs_zstd_cache_reap_now(void)
```

Reply:
done